core/common/src/thrift/block_master.thrift

```diff
@@ -56,7 +46,42 @@ struct GetUsedBytesTResponse {
   1: i64 bytes
 }
 
-struct GetWorkerInfoListTOptions {}
+enum WorkerRange {
+  ALL
+  LIVE
+  LOST
+  SPECIFIED
+}
+
+enum WorkerInfoField {
+  ADDRESS
+  CAPACITY_BYTES
+  CAPACITY_BYTES_ON_TIERS
+  ID
+  LAST_CONTACT_SEC
+  START_TIME_MS
+  USED_BYTES
+  USED_BYTES_ON_TIERS
+}
+
+struct WorkerInfo {
```

Comment:
if we remove fields, it will break backwards compatibility with previous versions. We need this to work when the client and server are different versions.

Reply:
@aaudiber The whole Report Command only works on the new version(with Alluxio cluster starts on the new Thrift server codes). The users use Report command will get nothing from the previous version thrift server. (Great different MasterInfo and WorkerInfo)

Reply:
The report command will work ok, but `WorkerInfo` was used in previous versions for the `getWorkerInfoList` RPC. Those versions will expect to see the fields. I'm not sure if `state` ever gets used, but `capacityBytes` is definitely needed. To be safe we should never delete fields from thrift structures, only add new fields
```
4: string state
5: i64 capacityBytes
```

Reply:
Add state and getWorkerInfoList() back to support different versions